frontend/packages/dev-console/src/components/import/pipeline/PipelineSection.tsx

```diff
@@ -15,7 +16,14 @@ type PipelineSectionProps = {
 const PipelineSection: React.FC<PipelineSectionProps> = ({ flags }) => {
   const { values } = useFormikContext<FormikValues>();
 
-  if (flags[FLAG_OPENSHIFT_PIPELINE]) {
+  const canListPiplines = useAccessReview({
+    group: PipelineModel.apiGroup,
+    resource: PipelineModel.plural,
+    namespace: 'openshift',
+    verb: 'list',
+  });
```

Comment:
At the end of this, the user will need to be able to create a new pipeline instance as well. Shouldn't we also ensure they can create the pipeline?

Reply:
@christianvogt We have not create pipeline option in add form.

Reply:
@vikram-raj There is a creation when you select `Add Pipeline` and [submit the form](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/import/import-submit-utils.ts#L423).

Reply:
Ah ok. I will add RBAC check for create pipeline as well.

Reply:
@christianvogt Added RBAC checks for create pipeline access.